### PR TITLE
Show last modified date on article pages

### PR DESCRIPTION
**Pull request by holder-josh**
Desktop:
expand

Mobile:
expandmobile

Articles which haven't been modified display as normal.
Also falls back gracefully for no-js users by leaving the date grey and not a link.

_NB: I still fundamentally believe this is a confusing feature for users. Something as small as a tag change or the addition of a pullquote will alert readers to the article being 'modified'. I feel that unexplained article modifications look suspicious to our readers and it seems at least 50% of articles are 'modified' in some way._

### CODE REVIEW

**Comment by paperboyo** [comment]
Hi,
Hi,

Wow, cool! We should also state the paper publication date and page when known.

Regards
Mateusz

**Comment by michaelwmcnamara** [comment]
Lovely idea! Are we expecting this to work for mobiles as well or is it just for desktop browsers?

**Comment by Jholder112233** [comment]
@michaelwmcnamara Should work on mobile too, I only tested it on my iPhone 5 but all worked as expected there.

**Comment by michaelwmcnamara** [comment]
Ace! Thanks Josh (did a little extra checkign and can confirm that it works on androids and shows as greyed out on older phones OS's too (ie Android 2.3).
+1

**Comment by Jholder112233** [comment]
@michaelwmcnamara thanks for testing that it falls back gracefully on old Android devices!

@mattosborn, +1 me?

File: static/src/javascripts/projects/common/modules/ui/last-modified.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+], function (
+    bean,
+    $
+) {
+    function expand() {
+        $('.js-lm').toggleClass('u-h');
+    }
+
+    function init() {
+        if ($('.js-lm').length !== 0) {
```
Comment (phamann) [inline static/src/javascripts/projects/common/modules/ui/last-modified.js]: You could just use `if ($('.js-lm').length)`

File: static/src/javascripts/projects/common/modules/ui/last-modified.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            $('.js-wpd').addClass('content__dateline-wpd--modified tone-colour');
+            bean.on(document.body, 'click', '.js-wpd', expand);
```
Comment (phamann) [inline static/src/javascripts/projects/common/modules/ui/last-modified.js]: Why are you binding to `document.body` here, surely the parent should suffice are their more than one?

File: static/src/stylesheets/module/content/_content.scss [hunk 3]
```diff
@@ -395,17 +399,28 @@
...
 
-    @include mq(wide) {
-        .content__dateline-time {
-            display: inline-block;
-        }
+.content__dateline-wpd--modified {
+    color: colour(news-default);
+    cursor: pointer;
+
+    &:hover {
```
Comment (phamann) [inline static/src/stylesheets/module/content/_content.scss]: And `:focus`,`:active`

**Comment by phamann** [comment]
I still think this interaction isn't clear enough to mobile users whom don't have hover...

**Comment by phamann** [comment]
:+1: Other than comments.

**Comment by paperboyo** [comment]
Hi,

Super minor point - the active area stretches the whole row:

URL: http://www.theguardian.com/stage/ng-interactive/2015/feb/17/can-you-identify-the-16-shows-up-for-the-oliviers-audience-award

modified

Regards
Mateusz

**Comment by Jholder112233** [comment]
@paperboyo good spot, on it!

**Pull request merged**